charts/internal/machine-controller-manager/seed/templates/vpa.yaml

```diff
@@ -17,10 +17,16 @@ spec:
       minAllowed:
         cpu: {{ .Values.resources.mcmProviderAlicloud.requests.cpu }}
         memory: {{ .Values.resources.mcmProviderAlicloud.requests.memory }}
+      maxAllowed:
+        cpu: {{.Values.resources.mcmProviderAlicloud.maxAllowed.cpu}}
+        memory: {{.Values.resources.mcmProviderAlicloud.maxAllowed.memory}}
```

Comment:
there are more places where vpa is used, do you also need to consider them?
https://github.com/gardener/gardener-extension-provider-alicloud/blob/master/charts/internal/seed-controlplane/charts/csi-alicloud/templates/csi-snapshot-controller-vpa.yaml
https://github.com/gardener/gardener-extension-provider-alicloud/blob/master/charts/internal/seed-controlplane/charts/csi-alicloud/templates/csi-snapshot-validation-webhook-vpa.yaml
https://github.com/gardener/gardener-extension-provider-alicloud/tree/master/charts/internal/shoot-system-components/charts/csi-alicloud/templates
https://github.com/gardener/gardener-extension-provider-alicloud/blob/master/charts/internal/seed-controlplane/charts/csi-alicloud/templates/csi-plugin-controller-vpa.yaml
https://github.com/gardener/gardener-extension-provider-alicloud/blob/master/charts/gardener-extension-provider-alicloud/templates/vpa.yaml
https://github.com/gardener/gardener-extension-provider-alicloud/blob/master/charts/gardener-extension-admission-alicloud/charts/runtime/templates/vpa.yaml

Reply:
At the moments maxAllowed is set only on critical VPA objects: cloud-controller-manager, machine-controller-manager, csi-driver-controller and csi-driver-node